GenumKit/Parsers/ColorsFileParser.swift

```diff
@@ -52,53 +52,53 @@ private func parseHexString(hexString: String) throws -> UInt32 {
 // MARK: - Text File Parser
 
 public final class ColorsTextFileParser: ColorsFileParser {
-  public private(set) var colors = [String:UInt32]()
+  public fileprivate(set) var colors = [String:UInt32]()
```

Comment:
agree, I'll do a full audit. A lot of these I think are code style between `extension` grouping.